task_processing/executors/task_executor.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import abc
+from collections import namedtuple
+
+import six
+
+TaskConfig = namedtuple(
+    'TaskConfig',
+    ['image', 'cmd', 'cpus', 'mem', 'disk', 'volumes', 'ports', 'cap_add',
+     'ulimit', 'docker_parameters'],
+)
+
+
+def make_task_config(image="ubuntu:xenial", cmd="/bin/true", cpus=0.1,
```

Comment:
If possible I'd like to avoid doing what paasta has done with method -> type constructors. Can we re-use an existing solution like https://github.com/python-attrs/attrs or https://pypi.python.org/pypi/namedlist/1.4?